python/semantic_kernel/processes/kernel_process/kernel_process_metadata_types.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from typing import Annotated
+
+from pydantic import Field
+
+from semantic_kernel.processes.kernel_process.kernel_process_state_metadata import KernelProcessStateMetadata
+from semantic_kernel.processes.kernel_process.kernel_process_step_state_metadata import KernelProcessStepStateMetadata
+
+process_metadata = Annotated[
```

Comment:
FYI: when using this is the type, they do not need to be based on the same base class, and you could also remove type from the base class and make that an ABC

Reply:
This was left in, and should have been removed. It's gone now.